src/benchmarks/micro/corefx/System.Memory/Slice.cs

```diff
@@ -152,7 +152,75 @@ public System.ReadOnlyMemory<T> ReadOnlyMemoryStartLength()
 
             return memory;
         }
-        
+
+        [Benchmark(OperationsPerInvoke = 16)]
+        public System.Memory<T> MemorySpanStart()
+        {
+            System.Memory<T> memory = new System.Memory<T>(_nonEmptyArray);
+
+            Consume(memory.Span.Slice(Size / 2)); Consume(memory.Span.Slice(Size / 2));
```

Comment:
Part of this test (initially) was to compare mem.Slice.Span and mem.Span.Slice. Can we retain that?
https://github.com/dotnet/corefx/blob/fd88c5df4bde4ef56522a1cd81da1ff69c92edc4/src/System.Memory/tests/Performance/Perf.Memory.Slice.cs#L45-L71

Or is this test porting something else?